docs/en_US/Release.md

```diff
@@ -1,5 +1,109 @@
 # ChangeLog
 
+## Release 1.2 - 12/02/2019
+
+### Major Features
+* Feature Engineering
+  - New feature engineering interface
+  - Feature selection algorithms: Gradient feature selector & GBDT selector
+* NAS
+  - New NAS interface
+  - NAS algorithms: ENAS, DARTS, P-DARTS (in PyTorch)
+  - NAS in classic mode (each trial runs independently)
+* Model compression
+  - New model pruning algorithms: lottery ticket pruning approach, L1Filter pruner, Slim pruner, FPGM pruner
+  - New model quantization algorithms: QAT quantizer, DoReFa quantizer
+* Training Service
+Support OpenPAI token authentication
+* Examples:
+  - A new MNIST trial example supports tensorflow 2.0.
+* Engineering Improvements
+  - For remote training service,  trial jobs require no GPU are now scheduled with round-robin policy instead of random.
+* Web Portal & User Experience
+  - Support user to add customized trial.
+  - User can zoom out/in in detail graphs, except Hyper-parameter.
+
+### Bug fix
+  - Fix the table sort issue when failed trials haven't metrics. -Issue #1773
+  - Maintain selected status(Maximal/Minimal) when the page switched. -PR#1710
+  - Make hyper-parameters graph's default metric yAxis more accurate. -PR#1736
```

Comment:
could add GPU related bug fix: permission, scheduling in remote mode

Reply:
Thandks, added.